debugProtocol.json

```diff
@@ -457,6 +457,10 @@
 										"Debugger attached to an existing process.",
 										"A project launcher component has launched a new process in a suspended state and then asked the debugger to attach."
 									]
+								},
+								"pointerLength": {
+									"type": "integer",
+									"description": "The size of a pointer or address for this process, in bits.  This value may be used by clients when formatting addresses for display."
```

Comment:
I think `pointerLength` description should say what the units are. Bits or Bytes.

Reply:
the description already says: "The size of a pointer or address for this process, in bits.."